tools/crm_mon.c

```diff
@@ -1475,8 +1560,10 @@ main(int argc, char **argv)
     do {
         print_as(output_format ,"Waiting until cluster is available on this node ...\n");
 
-        fencing_connect();
-        rc = cib_connect(TRUE);
+        if ((rc = pacemakerd_status()) == pcmk_rc_ok) {
```

Comment:
I think Ken just pushed an update to the coding standards suggesting the above be written like this instead:
```
rc = pacemakerd_status();
if (rc == pcmk_rc_ok) { ...
```

Reply:
Yep the goal of this one is when reading the code, it's immediately obvious, especially to coders of different skill levels. But it is purely stylistic ...